source/logs/command-history.txt

```diff
@@ -32,3 +32,36 @@ text editor:
 
    * - Windows
      - ``%UserProfile%/.mongodb/mongosh/mongosh_repl_history``
+
+Starting in |mdb-shell| 2.4.0, you can use the ``history()`` command to
+view a list of previously executed commands. For example:
+
+.. code-block:: javascript
+
+   history()
+
+Example output, which shows a list of commands in an array of strings:
+
+.. code-block:: javascript
+   :copyable: false
+
+   [
+     'db.salesFromFile.find()',
+     '   let salesCollection = JSON.parse( db.sales.find().toArray() )   ',
```

Comment:
[question] When I ran this sequence of commands, the `history()` output doesn't have these indentations. Did you add these for readability?

Reply:
No, didn't add anything. It adds the indents depending on what commands were run. Will leave as is. Cheers.

Reply:
Sounds good, thanks for the clarification!